webapp/src/main/webapp/src/app/sender/archive/sender-archive.component.ts

```diff
@@ -0,0 +1,68 @@
+import { Component, OnInit } from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import { Subscription } from 'rxjs';
+import { BaseListingComponent } from 'src/app/shared/base-listing.component';
+import { SenderSyncArchive } from './sender-archive';
+import { SenderArchiveService } from './sender-archive.service';
+import { SenderSyncArchiveCountAndItems } from './sender-sync-archive-count-and-items';
+import { SenderArchivedLoaded} from './state/sender-archive.actions';
+import { GET_SYNC_ARCHIVE } from './state/sender-archive.reducer';
+
+@Component({
+	selector: 'app-sender-archive',
+	templateUrl: './sender-archive.component.html',
+})
+export class SenderArchiveComponent extends BaseListingComponent implements OnInit {
+
+	count?: number;
+
+	loadedSubscription?: Subscription;
+
+	senderArchiveItens?: SenderSyncArchive[];
+
+	startDate?: string;
+
+	endDate?: string;
+
+	constructor(private service: SenderArchiveService,
+		private store: Store) {
+		super();
+	}
+
+	ngOnInit() {
+		this.init();
+
+		this.loadedSubscription = this.store.pipe(select(GET_SYNC_ARCHIVE)).subscribe(
+			countAndItems => {
+				this.count = countAndItems?.count;
+				this.senderArchiveItens = countAndItems?.items
```

Comment:
And here too, you nolonger need the `?`

Reply:
Is forcing me to put a '?' even with the set

Reply:
Pleaser refer to other components, we are not doing this, may be there is something not properly setup in the component

Reply:
Done